src/main/js/samples.js

```diff
@@ -67,10 +67,12 @@ samples.push({
         "   }\n" +
         "   stage('Build') {\n" +
         "      // Run the maven build\n" +
-        "      if (isUnix()) {\n" +
-        "         sh \"'${mvnHome}/bin/mvn' -Dmaven.test.failure.ignore clean package\"\n" +
-        "      } else {\n" +
-        "         bat(/\"${mvnHome}\\bin\\mvn\" -Dmaven.test.failure.ignore clean package/)\n" +
+        "      withEnv([\"MVN_HOME=$mvnHome\"]) {\n" +
+        "         if (isUnix()) {\n" +
+        "            sh '\"$MVN_HOME/bin/mvn\" -Dmaven.test.failure.ignore clean package'\n" +
+        "         } else {\n" +
+        "            bat(/\"%MVN_HOME%\\bin\\mvn\" -Dmaven.test.failure.ignore clean package/)\n" +
```

Comment:
Using "regex" string delimiters here seems weird. Why, other than that the old example did it? https://github.com/jenkinsci/workflow-cps-plugin/commit/b775c2aa77aa0c13390464f73e24dab9bf588c4f#diff-030263ebf957b3984b3dd6a035d5dfb2R73 doesn't explain.

Reply:
Makes it easier to use `/` on Windows: no need for an escape.